packages/@aws-cdk/aws-lambda/lib/function-url.ts

```diff
@@ -114,9 +114,17 @@ export interface IFunctionUrl extends IResource {
    * The url of the Lambda function.
    *
    * @attribute FunctionUrl
+   * @example `https://*******.lambda-url.ap-southeast-2.on.aws/`
```

Comment:
The example tagged under `@example` will be tested for compilation. This should just be "For example, ..."